facturacion_mexico/cfdi_recibidos/tests/test_wizard_manual_templates.py

```
@@ -0,0 +1,186 @@
+        cls.acc_iva = _get_or_create_tax_account(f"_Test IVA WMT {_H}", TEST_COMPANY)
+        cls.acc_iva8 = _get_or_create_tax_account(f"_Test IVA8 WMT {_H}", TEST_COMPANY)
+        cls.acc_ieps = _get_or_create_tax_account(f"_Test IEPS WMT {_H}", TEST_COMPANY)
+
+    @classmethod
+    def tearDownClass(cls):
+        _cleanup_templates(TEST_COMPANY, f"WMT {_H}")
+        for acc in [cls.acc_iva, cls.acc_iva8, cls.acc_ieps]:
+            try:
+                frappe.delete_doc("Account", acc, force=True)
+            except Exception:
+                pass
```

Ruff (0.15.15)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)